accessscopes.go

```diff
@@ -0,0 +1,28 @@
+package goshopify
+
+type AccessScopesService interface {
+	Get(interface{}) ([]AccessScope, error)
+}
+
+type AccessScope struct {
+	Handle string `json:"handle,omitempty"`
+}
+
+// AccessScopesResource represents the result from the oauth/access_scopes.json endpoint
+type AccessScopesResource struct {
+	AccessScopes []AccessScope `json:"access_scopes,omitempty"`
+}
+
+// AccessScopesServiceOp handles communication with the Access Scopes
+// related methods of the Shopify API.
+type AccessScopesServiceOp struct {
+	client     *Client
+}
+
+// Get access scopes based on used oauth token
+func (s *AccessScopesServiceOp) Get(options interface{}) ([]AccessScope, error) {
```

Comment:
Should this be list since it returns an array?

Reply:
Yep, updated accordingly, thanks!